plugins/rildev.c

```diff
@@ -124,11 +296,14 @@ static int detect_init(void)
 			num_slots = env_slots;
 	}
 
-	ofono_info("RILDEV detected modem type %s, %d SIM slot(s)",
-			ril_type, num_slots);
+	ofono_info("RILDEV searching modem type %s, %d SIM slot(s)",
+							ril_type, num_slots);
+
+	/* Add watch as sockets might take a while to create */
+	start_inotify();
```

Comment:
Why does the code ignore the retval from start_inotify?  If it's not going to be used, please make the function specify void for it's retval.